docs/tutorial/setting-up/index.md

```diff
@@ -48,6 +48,19 @@ In this section, you will run an OpenSAFELY command to test that the codespace i
 
 Towards the bottom of the codespace, you will see the terminal.
 
+??? tip "The terminal"
+    The terminal allows you to interact with programs,
+    such as ehrQL,
+    that have command line interfaces.
+    Whilst *terminal* and *shell* are often used synonymously,
+    a shell runs within a terminal.
+
+    You may have used a shell before:
+    Git Bash, for example, bundles Git with the Bash shell for the Windows operating system.
+    The codespace's default shell is also the Bash shell,
+    so if you've used Git Bash before,
+    then you'll find the codespace's default shell familiar.
```

Comment:
Applying Dillingham's razor: do we really need this?

Reply:
Rose noticed that the shell was Bash and not Git Bash. Whilst `n=1` is a terrible sample size, our `n` is never going to be a large number. I think we need this, if only because Bash behaves slightly differently to Git Bash. For example, in some Git Bashes I've used, tab-completing a command adds an extension to the command: `column` becomes `column.exe`.